### ISSUE TITLE
Write policy for starting/stopping/restarting crond

### ISSUE THREAD

**Issue by ykhodorkovskiy** [TresysTechnology/clip#195]
```
type=USER_AVC msg=audit(1438549098.961:1956): pid=1 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:init_t:s0 msg='avc:  denied  { start } for auid=1000 uid=0 gid=0 path="/usr/lib/systemd/system/crond.service" cmdline="systemctl start crond.service" scontext=toor_u:toor_r:toor_t:s0 tcontext=system_u:object_r:systemd_unit_file_t:s0 tclass=service  exe="/usr/lib/systemd/systemd" sauid=0 hostname=? addr=? terminal=?'
type=AVC msg=audit(1438549099.271:1958): avc:  denied  { rlimitinh } for  pid=14297 comm="crond" scontext=system_u:system_r:init_t:s0 tcontext=system_u:system_r:crond_t:s0-s0:c0.c1023 tclass=process
type=AVC msg=audit(1438549099.271:1958): avc:  denied  { siginh } for  pid=14297 comm="crond" scontext=system_u:system_r:init_t:s0 tcontext=system_u:system_r:crond_t:s0-s0:c0.c1023 tclass=process
type=AVC msg=audit(1438549099.271:1958): avc:  denied  { noatsecure } for  pid=14297 comm="crond" scontext=system_u:system_r:init_t:s0 tcontext=system_u:system_r:crond_t:s0-s0:c0.c1023 tclass=process
type=USER_AVC msg=audit(1438549074.223:1953): pid=1 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:init_t:s0 msg='avc:  denied  { stop } for auid=1000 uid=0 gid=0 path="/usr/lib/systemd/system/crond.service" cmdline="systemctl stop crond.service" scontext=toor_u:toor_r:toor_t:s0 tcontext=system_u:object_r:systemd_unit_file_t:s0 tclass=service  exe="/usr/lib/systemd/systemd" sauid=0 hostname=? addr=? terminal=?'
type=USER_AVC msg=audit(1438549074.228:1955): pid=1 uid=0 auid=4294967295 ses=4294967295 subj=system_u:system_r:init_t:s0 msg='avc:  denied  { status } for auid=1000 uid=0 gid=0 path="/usr/lib/systemd/system/crond.service" cmdline="systemctl stop crond.service" scontext=toor_u:toor_r:toor_t:s0 tcontext=system_u:object_r:systemd_unit_file_t:s0 tclass=service  exe="/usr/lib/systemd/systemd" sauid=0 hostname=? addr=? terminal=?'
type=AVC msg=audit(1438545661.853:1544): avc:  denied  { search } for  pid=13550 comm="crond" name="faillock" dev="tmpfs" ino=14063 scontext=system_u:system_r:crond_t:s0-s0:c0.c1023 tcontext=system_u:object_r:faillog_t:s0 tclass=dir
type=AVC msg=audit(1438545661.861:1548): avc:  denied  { use } for  pid=13550 comm="crond" path="/run/systemd/sessions/10.ref" dev="tmpfs" ino=66942 scontext=system_u:system_r:crond_t:s0-s0:c0.c1023 tcontext=system_u:system_r:systemd_logind_t:s0 tclass=fd
type=AVC msg=audit(1438545661.861:1548): avc:  denied  { write } for  pid=13550 comm="crond" path="/run/systemd/sessions/10.ref" dev="tmpfs" ino=66942 scontext=system_u:system_r:crond_t:s0-s0:c0.c1023 tcontext=system_u:object_r:systemd_logind_var_run_t:s0 tclass=fifo_file```

**Comment by mpalmi** [comment]
Resolved with Issue #134 in https://github.com/mpalmi/clip/commit/64322f04c002c00f5e6f71e09886de8565b5e632

**Comment by ykhodorkovskiy** [comment]
Verified that this was resolved with #134. Leaving this open until #134 is merged.
